docs/output.md

```diff
@@ -0,0 +1,16 @@
+# nf-core/deepvariant: Output
+
+This document describes the output produced by the pipeline.
+
+## VCF
+
+The output from DeepVariant is a variant call file or vcf
```

Comment:
variant call format file

Reply:
Nice would be a cross ref to the format specification and the version, like [VCF 4.2](https://samtools.github.io/hts-specs/VCFv4.2.pdf)